src/views/GovernanceTaskList/TaskListCta.tsx

```diff
@@ -5,17 +5,18 @@ import { Button, Link as UswdsLink } from '@trussworks/react-uswds';
 
 import { isIntakeStarted } from 'data/systemIntake';
 import { attendGrbMeetingTag } from 'data/taskList';
-import { SystemIntakeForm } from 'types/systemIntake';
+import { GetSystemIntake_systemIntake as SystemIntake } from 'queries/types/GetSystemIntake';
 
 // CTA for Task List Intake Draft
-export const IntakeDraftCta = ({ intake }: { intake: SystemIntakeForm }) => {
-  switch (intake.status) {
+export const IntakeDraftCta = ({ intake }: { intake: SystemIntake }) => {
```

Comment:
This component and the next few only access a couple properties from the system intake that's passed in. We could pull those properties out into props and then deconstruct them from the intake that is passed in. That might make the code more flexible since we can pass anything in as long as it has those two properties.

```suggestion
export const IntakeDraftCta = ({
  id,
  status
}: {
  id?: string;
  status?: string;
}) => ```

Reply:
Yeah I was thinking about that. Down below we check `isIntakeStarted(intake)` to check if the intake has been started. If it has been started, the text changes from `Start` to `Continue`.

We still need the entire intake here, but I could see the argument that the abstraction is happening at the wrong level.